test/integer.test.js

```diff
@@ -149,3 +153,51 @@ test('should round interger object parameter ', t => {
   t.equal(output, '{"magic":5}')
   t.ok(validate(JSON.parse(output)), 'valid schema')
 })
+
+test('should not stringify a property if it does not exist', t => {
+  t.plan(2)
+
+  const schema = { type: 'object', properties: { magic: { type: 'integer' } } }
+  const validate = validator(schema)
+  const stringify = build(schema)
+  const output = stringify({})
+
+  t.equal(output, '{}')
+  t.ok(validate(JSON.parse(output)), 'valid schema')
+})
+
+test('should not stringify a property if it does not exist (rounding: ceil)', t => {
+  t.plan(2)
+
+  const schema = { type: 'object', properties: { magic: { type: 'integer' } } }
+  const validate = validator(schema)
+  const stringify = build(schema, { rounding: 'ceil' })
```

Comment:
Just as optimization, you could create a dynamic test that accept it as parameter to avoid the code repetition.

Such as

```

[ceil, floor].forEach(rounding => {
  test('should not stringify a property if it does not exist (rounding: )'+rounding, t => {
    // ...
  }
})